src/chatClient.cpp

```diff
@@ -320,6 +320,35 @@ bool Client::openDb(const std::string& sid)
                 ok = true;
                 KR_LOG_WARNING("Database version has been updated to %s", gDbSchemaVersionSuffix);
             }
+            else if (cachedVersionSuffix == "10" && (strcmp(gDbSchemaVersionSuffix, "11") == 0))
+            {
+                // Remove USER_ATTR_RSA_PUBKEY attr from cache
+                db.query("delete from userattrs where type = 64");
+
+                // Create temporary table and copy sendkeys content
+                db.query("CREATE TABLE tempkeys(chatid int64 not null, userid int64 not null, keyid int32 not null, key blob not null,ts int not null, UNIQUE(chatid, userid, keyid));");
```

Comment:
This way of moving `int64` values into records with `int32` values sounds risky. If someone had legacy key ids (which I hesitate), it could create issues. But given that none should have those keys at these days and it's already implemented... let's keep it. :)

Reply:
Ok